pkgs/development/compilers/zig/default.nix

```diff
@@ -18,6 +18,9 @@ stdenv.mkDerivation rec {
     hash = "sha256-x2c4c9RSrNWGqEngio4ArW7dJjW0gg+8nqBwPcR721k=";
   };
 
+  # Fix index out of bounds reading RPATH (cherry-picked from 0.10-dev)
```

Comment:
Can we fetchpatch this?

Reply:
As discussed above, no, due to conflict.